<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMonitorWgt</class>
 <widget class="QWidget" name="MainMonitorWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title" native="true">
     <widget class="TGFrame" name="midVideoFrame1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt1" native="true">
         <widget class="QLabel" name="labelImei1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TGFrame" name="midVideoFrame2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt2" native="true">
         <widget class="QLabel" name="labelImei2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TGFrame" name="midVideoFrame3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt3" native="true">
         <widget class="QLabel" name="labelImei3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TGFrame" name="midVideoFrame4" native="true">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt4" native="true">
         <widget class="QLabel" name="labelImei4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TGFrame" name="midVideoFrame5" native="true">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt5" native="true">
         <widget class="QLabel" name="labelImei5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TGFrame" name="midVideoFrame6" native="true">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>400</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="OpenGLWidget" name="playWgt6" native="true">
         <widget class="QLabel" name="labelImei6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="PageWgt" name="pageWgt" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>440</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TGFrame</class>
   <extends>QWidget</extends>
   <header>Component/TGFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header>Component/OpenglWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageWgt</class>
   <extends>QWidget</extends>
   <header>Component/PageWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
